docs/topics/install-olympia/index.rst

```diff
@@ -1,16 +1,14 @@
 ===============
-Install Olympia
+Server Install
 ===============
 
+
+
 This is a legacy section for installing Olympia manually (and painfully) by hand.
 **Instead, you'll find it easier to** :ref:`install Olympia from Docker <install-with-docker>`.
```

Comment:
I'm not sure this sentence is still relevant: it present two ways to install the server, using docker (the first method, the one you get when you click "next"), and the legacy one after that.

Maybe we could have something slightly different: remove the above sentence (or reword it to explain that the recommended way to install is via docker), and then explain below the toctree (that would only have the "docker" install) that there's also a legacy deprecated way to install it, and then link to `deprecated/index` there (without a toctree in this page to avoid confusion).